tests/run-make/arm64ec-import-export-static/export.rs

```diff
@@ -0,0 +1,27 @@
+#![crate_type = "dylib"]
+#![allow(internal_features)]
+#![feature(no_core, lang_items)]
+#![no_core]
+#![no_std]
+
+// This is needed because of #![no_core]:
+#[lang = "pointee_sized"]
+pub trait PointeeSized {}
+#[lang = "meta_sized"]
+pub trait MetaSized: PointeeSized {}
+#[lang = "sized"]
+pub trait Sized: MetaSized {}
+#[lang = "sync"]
+trait Sync {}
+impl Sync for i32 {}
+#[lang = "copy"]
+pub trait Copy {}
+impl Copy for i32 {}
+#[lang = "drop_in_place"]
+pub unsafe fn drop_in_place<T: ?Sized>(_: *mut T) {}
```

Comment:
Can this test use mini_core? Or does that not work for run-make tests?

Reply:
I have no idea: I've only seen `mini_core` used in the compiler tests. The tests in `run-make` use this pattern of manually declaring whatever they need.

Reply:
It probably isn't supported yet then.